src/yang/iana-qos-types.yang

```diff
@@ -61,325 +61,272 @@ module iana-qos-types {
+      "Action type specified as a named counter.";
   }
   identity queue-policy-name {
     base action-type;
     description
-      "queue policy name";
+      "Action type specified as a queue policy name.";
   }
   
   identity meter-type {
     description
-      "This base identity type defines meter types";
+      "Base identity for types of meter.";
   }
   identity one-rate-two-color {
     base meter-type;
     description
-      "one rate two color meter type";
+      "Meter type specified as one rate two color meter.";
   }
   identity one-rate-tri-color {
     base meter-type;
     description
-      "one rate three color meter type";
+      "Meter type specified as one rate three color meter.";
   }
   identity two-rate-tri-color {
     base meter-type;
     description
-      "two rate three color meter action type";
+      "Meter type specified as two rate three color meter.";
   }
 
   identity drop-type {
     description
-      "drop algorithm";
+      "Base identity for drop algorithm.";
   }
   identity tail-drop {
     base drop-type;
     description
-      "tail drop algorithm";
+      "Drop algorithm specified as tail drop.";
   }
   identity red {
     base drop-type;
     description
-      "Random Early Detect drop algorithm";
+      "Drop algorithm specified as RED.";
   }
   identity wred {
     base drop-type;
     description
-      "Weighted Random Early Detect drop algorithm";
+      "Drop algorithm specified as WRED.";
   }
 
   identity rate-unit-type {
     description
-      "base rate-unit type";
+      "Base definition for the type of rate unit.";
   }
   identity bits-per-second {
     base rate-unit-type;
     description
-      "bits per second identity";
-  }
-  identity kilo-bits-per-second {
-    base rate-unit-type;
-    description
-      "kilo bits per second identity";
-  }
-  identity mega-bits-per-second {
-    base rate-unit-type;
-    description
-      "mega bits per second identity";
-  }
-  identity giga-bits-per-second {
-    base rate-unit-type;
-    description
-      "mega bits per second identity";
+      "Rate specified in bits per second.";
   }
   identity percent {
     base rate-unit-type;
     description
-      "percentage";
+      "Rate specified as a percentage of queue depth.";
   }
 
   identity burst-unit-type {
     description
-      "base burst-unit type";
+      "Base definition for burst specification.";
   }
   identity bytes {
     base burst-unit-type;
     description
-      "bytes";
-  }
-  identity kilo-bytes {
-    base burst-unit-type;
-    description
-      "kilo bytes";
-  }
-  identity mega-bytes {
-    base burst-unit-type;
-    description
-      "mega bytes";
+      "Burst specified in bytes.";
   }
   identity millisecond {
     base burst-unit-type;
     description
-      "milli seconds";
-  }
-  identity microsecond {
-    base burst-unit-type;
-    description
-      "micro seconds";
+      "Burst specified in milliseconds.";
```

Comment:
not sure if microsecond removal will be good idea.. for 100 Gbps, 1 millisecond will translate to 12 Mbytes of burst.. which could be more than someone asking..

Reply:
But your burst value is specified as a uint64. Therefore if someone has to represent 12Mbytes of burst its value is 12582912 bytes, which the value parameter can still take.

Reply:
I see. I will change the minimum unit to microsecond, instead of millisecond.